components/forecast/AvalancheForecast.tsx

```diff
@@ -126,7 +126,7 @@ export const AvalancheForecast: React.FunctionComponent<AvalancheForecastProps>
           <AvalancheTab zone={zone} forecast={forecast} />
         </Tab>
         <Tab title="Weather">
-          <WeatherTab zone={zone} />
+          <WeatherTab zone={zone} center_id={center_id} date={date} />
```

Comment:
Still kind of weird that we're not passing a zone here, the screen is very specifically for the zone we're looking at the avy forecast for, right?

Reply:
Isn't `zone` the very first param there? 😕

Reply:
:facepalm: